source/cxx.txt

```diff
@@ -68,7 +68,7 @@ MongoDB Compatibility
 .. admonition:: Development
    :class: important
 
-   `*` denotes a version in development.
+   ``*`` denotes a version in development.
```

Comment:
Should we keep this text if none of the displayed driver versions are in development?